source/context_menus/_context_menu_stats.js

```diff
@@ -0,0 +1,58 @@
+const { PermissionFlagBits, InteractionContextType } = require('discord.js');
+const { UserContextMenuWrapper } = require('../classes');
+
+const mainId = "User Stats";
```

Comment:
"BountyBot Stats" or "Profile Stats" might be more clear which context the stats are from.

Reply:
As much as I'm leery about beginning to attach "BountyBot" to all our context menus, I think "Profile" is the same level of confusing as "User" here. "BountyBot" it is.

Reply:
I'm also not a fan of an unconditional "BountyBot" prefix (I don't think it'll end up bleeding into too many more, but I guess we'll see on that one?).